File: mario-more/mario.c
// Prints right-aligned pyramid
#include <cs50.h>
#include <stdio.h>

void print_row(int spaces, int bricks);

int main(void)
{
    // Promt the user for pyramid height
    int height;
    do
    {
        height = get_int("Please enter the height: ");
    }
    while (height < 1);

    // prints the pyramid with given height
    for (int r = 0; r < height; r++)
    {
        print_row(height - (r + 1), r + 1);
    }
}

// Print row of bricks(right aligned)
void print_row(int spaces, int bricks)
{
    // Print left spaces
    for (int j = 0; j < spaces; j++)
    {
        printf(" ");
    }

    // Print left bricks
    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }

    // Print centre spaces
    printf("  ");

    // Print right bricks
    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }

    printf("\n")

}

File: aaaaa/CS50x/pset6/sentimental-mario-more/mario.py
# Program to print

# Funtion import
from cs50 import get_int


def main():

    # Promts user for height
    while True:
        height = get_int("Enter Height: ")
        if height >= 1 and height <= 8:
            break

    # Prints the pyramid with given height
    for r in range(height):
        print_row(height - (r + 1), r + 1)


# Print row
def print_row(spaces, bricks):

    # Print left spaces
    for s in range(spaces):
        print(" ", end="")

    # Print left bricks
    for b in range(bricks):
        print("#", end="")

    # Print centre spaces
    print("  ", end="")

    # Print right bricks
    for b in range(bricks):
        print("#", end="")

    # Move to next line
    print("")


main()
